.changeset/2ec66e64/changes.json

```diff
@@ -1 +1 @@
-{ "releases": [{ "name": "@keystone-alpha/fields", "type": "patch" }], "dependents": [] }
+{ "releases": [{ "name": "@keystone-alpha/fields", "type": "major" }], "dependents": [] }
```

Comment:
I realized this was incorrectly marked as `patch` when it's a `major` change (#1014)